NewDawn/NewDawn/ProfileViewController/Profile_DraftFinal.swift

```diff
@@ -251,4 +256,28 @@ class Profile_DraftFinal: UIViewController {
         }
     }
     
+    func notificationSetUp() -> Void{
+        // Push notification with instance id and connect to "interest"
+        pushNotifications.start(instanceId: "466cb5de-0fd2-40c7-ad45-802b6c79550e")
+        pushNotifications.registerForRemoteNotifications()
+        try? pushNotifications.addDeviceInterest(interest: "hello")
```

Comment:
What does hello mean?

Reply:
That's just another way of sending notification like a broadcast, instead of user-specific. It works without this line.

Reply:
Maybe let's change this to a name that better indicate what we want to broadcast, like "daily_recommendation"?